packages/lib-react-components/stories/Media.stories.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
```

Comment:
Just a couple of general thoughts on storybooks:

- should we be moving them to their actual components?
- should we be adding knobs for these props?
- should we remove the story source? (I'm not finding it very helpful)

Reply:
Yeah, we could use a better setup. The separate folder is what the cli generates for you when you first get started. Could you open an issue for it?